packages/editor/src/components/browser/TypeBrowser.tsx

```diff
@@ -137,7 +196,7 @@ const TypeBrowser = (props: { value: string; onChange: (value: string) => void;
           }
         }}
       >
-        {mainFilter.length > 0 ? (
+        {tableDynamic.getRowModel().rows.length > 0 ? (
           <tbody>

```

Comment:
you can also deduplicate the tbody by move this out of the condition. maybe then you need to use a <> fragment inside